tiflash/troubleshoot-tiflash.md

```diff
@@ -97,3 +98,168 @@ Take the following steps to handle the data file corruption:
 1. Refer to [Take a TiFlash node down](/scale-tidb-using-tiup.md#scale-in-a-tiflash-cluster) to take the corresponding TiFlash node down.
 2. Delete the related data of the TiFlash node.
 3. Redeploy the TiFlash node in the cluster.
+
+## TiFlash analysis is slow
+
+If a statement contains operators or functions not supported in MPP mode, TiDB does not select MPP mode. This is because the analysis of the statement is slow. In this case, you can execute the `EXPLAIN` statement to check for operators or functions not supported in MPP mode.
+
+{{< copyable "sql" >}}
+
+```sql
+create table t(a datetime);
+alter table t set tiflash replica 1;
+insert into t values('2022-01-13');
+set @@session.tidb_enforce_mpp=1;
+explain select count(*) from t where subtime(a, '12:00:00') > '2022-01-01' group by a;
+show warnings;
+```
+
+In this example, the warning message shows that TiDB does not select MPP mode because TiDB 5.4 and earlier versions do not support the `subtime` function.
+
+```
++---------+------+-----------------------------------------------------------------------------+
+> | Level   | Code | Message                                                                     |
++---------+------+-----------------------------------------------------------------------------+
+| Warning | 1105 | Scalar function 'subtime'(signature: SubDatetimeAndString, return type: datetime) is not supported to push down to tiflash now.       |
++---------+------+-----------------------------------------------------------------------------+
+```
+
+## Data is not replicated to TiFlash
+
+After deploying a TiFlash node and starting replication (by performing the ALTER operation), no data is replicated to it. You can identify and address the problem by following the steps below:
+
+1. Check whether the replication is successful by running the `ALTER table <tbl_name> set tiflash replica <num>` command and check the output.
+
+    - If there is output, go to the next step.
+    - If there is no output, run the `SELECT * FROM information_schema.tiflash_replica` command to check whether TiFlash replicas have been created. If not, run the `ALTER table ${tbl_name} set tiflash replica ${num}` command again, check whether other statements have been executed, or check whether DDL executions are successful.
+
+2. Check whether the TiFlash process runs correctly.
+
+   Check whether there is any change in `progress`, the `flash_region_count` parameter in the `tiflash_cluster_manager.log` file, and `Uptime`:
```

Comment:
Is `Uptime` a parameter in `tiflash_cluster_manager.log`?

Reply:
> Is `Uptime` a parameter in `tiflash_cluster_manager.log`?

No. It's a monitoring item on Grafana. Maybe the sentence can be changed to "Check whether there is any change in `progress`, the `flash_region_count` parameter in the `tiflash_cluster_manager.log` file, and the Grafana monitoring item `Uptime`.